tests/unit_tests/test_wave_03.py

```diff
@@ -131,7 +131,8 @@ def test_swap_items_from_their_empty_returns_false():
 
     result = fatimah.swap_items(jolie, item_b, nobodys_item)
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert result is False
+    assert len(fatimah.inventory) == 3
+    assert len(jolie.inventory) == 0
+    assert fatimah.inventory == [item_a, item_b, item_c]
+    assert jolie.inventory == []
```

Comment:
👍 Since we didn't expect a swap to occur, it's reasonable to assume that items are in the same position they originally started. However, it might still be preferred to use a length check with individual `in` checks for the items rather than enforcing this unstated assumption.

However, using list comparison means that we could skip checking the length separately, since lists are only equal if they have the same length.